utilities_test.go

```diff
@@ -96,7 +97,86 @@ func TestDatasetWarp(t *testing.T) {
 		t.Errorf("wrong block size %d,%d", st.BlockSizeX, st.BlockSizeY)
 	}
 }
+func TestDatasetWarpMulti(t *testing.T) {
+	ds1, _ := Open("testdata/test_1_1.tif", RasterOnly())
```

Comment:
Lets try to limit the number of test files in the repo. Please make ds1..N dynamically with ```Create(Mem, "", ...)```